torch/testing/_internal/distributed/rpc/jit/dist_autograd_test.py

```diff
@@ -3,16 +3,30 @@
 import torch
 import torch.distributed.autograd as dist_autograd
 from torch.testing import FileCheck
-from torch.testing._internal.dist_utils import dist_init
+from torch.testing._internal.dist_utils import dist_init, worker_name
 from torch.testing._internal.distributed.rpc.rpc_agent_test_fixture import (
     RpcAgentTestFixture,
 )
+from torch.distributed.rpc import rpc_async
 
+@torch.jit.script
+def local_add(t1, t2):
+    return torch.add(t1, t2)
+
+@torch.jit.script
+def remote_add(t1, t2, dst: str):
```

Comment:
To silent linter, need to annotate `def remote_add(t1, t2, dst: str):  # noqa: E999`